### PR TITLE
[docs] Update Deckhouse Virtualization Platform documentation

### PR DESCRIPTION
**Pull request by mortelumina98**
## Description

Update Deckhouse Virtualization Platform documentation.

## Changelog entries
```changes
section: docs
type: chore
summary: Update Deckhouse Virtualization Platform documentation
impact_level: low
```

### CODE REVIEW

File: docs/site/pages/virtualization-platform/documentation/user/resource-managment/DISKS.md [hunk 1]
```diff
@@ -3,4 +3,210 @@ title: "Disks"
...
+
+```console
+# NAME                          PROVISIONER                           RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+# i-linstor-thin-r1 (default)   replicated.csi.storage.deckhouse.io   Delete          Immediate              true                   48d
+# i-linstor-thin-r2             replicated.csi.storage.deckhouse.io   Delete          Immediate              true                   48d
+# i-linstor-thin-r3             replicated.csi.storage.deckhouse.io   Delete          Immediate              true                   48d
+# linstor-thin-r1               replicated.csi.storage.deckhouse.io   Delete          WaitForFirstConsumer   true                   48d
+# linstor-thin-r2               replicated.csi.storage.deckhouse.io   Delete          WaitForFirstConsumer   true                   48d
+# linstor-thin-r3               replicated.csi.storage.deckhouse.io   Delete          WaitForFirstConsumer   true                   48d
+# nfs-4-1-wffc                  nfs.csi.k8s.io                        Delete          WaitForFirstConsumer   true                   30d
```
Comment (z9r5) [inline docs/site/pages/virtualization-platform/documentation/user/resource-managment/DISKS.md:48]:
```suggestion
# NAME                          PROVISIONER                           RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
# i-linstor-thin-r1 (default)   replicated.csi.storage.deckhouse.io   Delete          Immediate              true                   48d
# i-linstor-thin-r2             replicated.csi.storage.deckhouse.io   Delete          Immediate              true                   48d
# i-linstor-thin-r3             replicated.csi.storage.deckhouse.io   Delete          Immediate              true                   48d
# linstor-thin-r1               replicated.csi.storage.deckhouse.io   Delete          WaitForFirstConsumer   true                   48d
# linstor-thin-r2               replicated.csi.storage.deckhouse.io   Delete          WaitForFirstConsumer   true                   48d
# linstor-thin-r3               replicated.csi.storage.deckhouse.io   Delete          WaitForFirstConsumer   true                   48d
# nfs-4-1-wffc                  nfs.csi.k8s.io                        Delete          WaitForFirstConsumer   true                   30d
```

Решетки надо убрать. В выводе их нет. @mortelumina98

File: docs/site/pages/virtualization-platform/documentation/user/resource-managment/IMAGES.md [hunk 2]
```diff
@@ -3,4 +3,266 @@ title: "Images"
...
+
+The image creation process involves the following steps:
+
+1. The user creates a [VirtualImage](../../../reference/cr/virtualimage.html) resource.
+1. Once created, the image is automatically downloaded from the specified source in the specification to the storage (DVCR).
+1. After the download is complete, the resource becomes available for disk creation.
+
+There are different types of images:
+
+- ISO Image: An installation image used for the initial installation of an operating system. Such images are released by OS vendors and are used for installation on physical and virtual servers.
```
Comment (Lada7878) [inline docs/site/pages/virtualization-platform/documentation/user/resource-managment/IMAGES.md:16]:
```suggestion
- ISO Image: An installation image used for the initial installation of an operating system. Such images are released by OS vendors and are used for installation on physical and virtual servers.
```

Maybe: An image used for the initial installation? Same in russian.

File: docs/site/pages/virtualization-platform/documentation/user/resource-managment/IMAGES.md [hunk 3]
```diff
@@ -3,4 +3,266 @@ title: "Images"
...
+- Preinstalled System Disk Image: Contains a pre-installed and configured operating system, ready for use after creating a virtual machine. These images are offered by various vendors and can come in formats such as qcow2, raw, vmdk, and others.
```
Comment (Lada7878) [inline docs/site/pages/virtualization-platform/documentation/user/resource-managment/IMAGES.md:17]:
```suggestion
- Preinstalled System Disk Image: Contains configured operating system, ready for use after creating a virtual machine. These images are offered by various vendors and can come in formats such as qcow2, raw, vmdk, and others.
```

How about that? And russian

File: docs/site/pages/virtualization-platform/documentation/user/resource-managment/IMAGES.md [hunk 4]
```diff
@@ -3,4 +3,266 @@ title: "Images"
...
+After creating the resource, the type and size of the image are automatically determined, and this information is reflected in the resource's status.
+
+Images can be uploaded from various sources, such as HTTP servers where the image files are hosted, or container registries. Additionally, images can be uploaded directly from the command line using the `curl` utility.
+
+Images can also be created from other images and virtual machine disks.
+
+For project-specific images, two storage options are supported:
+
+- Container registry: The default type where the image is stored in the DVCR.
+- Persistent Volume Claim`: This type uses PVC as the storage for the image. This option is preferable when using storage that supports fast PVC cloning, as disk creation from images will be faster in this case.
```
Comment (Lada7878) [inline docs/site/pages/virtualization-platform/documentation/user/resource-managment/IMAGES.md:34]:
```suggestion
- Persistent Volume Claim: This type uses PVC as the storage for the image. This option is preferable when using storage that supports fast PVC cloning, as disk creation from images will be faster in this case.
```

**Review by Lada7878 — APPROVED** [review]

**Pull request merged**